experiments/AtmosLES/stable_bl_les.jl

```diff
@@ -11,6 +15,11 @@ function main()
         metavar = "prescribed|bulk"
         arg_type = String
         default = "bulk"
+
+        "--CP-Version"
```

Comment:
I think this should specify the path to the definition file rather than a specific version, the you are not reliant on a specific formatting assumption (name of the file and how you are formatting it) in the CES driver